lib/recurly/purchase.php

```diff
@@ -7,6 +7,9 @@
  * @property string $currency The currency to use in this invoice
  * @property string $po_number The po number for the invoice
  * @property integer $net_terms The net terms of the invoice
+ * @property array $coupon_codes An array of coupon codes to apply to the purchase
+ * @property array $subscriptions An array of subscriptions to apply to the purchase
```

Comment:
Looking at the [PHPDoc docs for types](https://phpdoc.org/docs/latest/references/phpdoc/types.html), it seems like we can document the types in the array using:
```
 * @property string[] $coupon_codes An array of coupon codes to apply to the purchase
 * @property Recurly_Subscription[] $subscriptions An array of subscriptions to apply to the purchase
```

Also, I'm not sure apply is the right verb to use for subscriptions.